specs/subresourceintegrity/spec.markdown

```diff
@@ -622,26 +622,24 @@ the element:
 <section>
 ###### The `script` element
 
-When executing step 5 of step 14 of HTML5's
-["prepare a script" algorithm][prepare]:
+Replace step 14.1 of HTML5's ["prepare a script" algorithm][prepare] with:
 
-1.  Set the [integrity metadata][] of the request to the value
-    of the element's `integrity` attribute.
+1.  Let <var>src</var> be the value of the element's `src` attribute and
+    <var>integrity</var> be the value of the element's `integrity` attribute.
```

Comment:
Don't you need a "Set the request's associated integrity metadata to be the value of the integrity attribute" ?